toolkits/global/packages/global-skip-link/README.md

```diff
@@ -17,40 +30,53 @@ To include `global-skip-link` in your application, you need to choose **ONE** br
 @import '@springernature/global-skip-link/scss/50-components/skip-link';
 ```
 
-## Usage
+### Link
 
-The skip-link is the first focusable element on the page.
-It is an <a> pointing to an id of the section it links to.
-For example if the main container is:
+The skip link should be a link to the main container of the page. You must supply an id for the link to point to.
+
+For example, if you supply `main` as the `id`:
 
 ```html
 <main id="main">
 ...
 </main>
 ```
 
-then the skip link should look like this:
+then the link would be:
 
 ```html
 <a class="c-skip-link" href="#main">
     Skip to main content
 </a>
 ```
 
-ATTENTION:
-1. The target section must be focusable.
-You will need to add a negative tabindex in order to improve screen reader accessibility for browsers like IE11.
-It's advised to do so via [JavaScript](https://codepen.io/200ok/pen/jvNBMP) and not directly in the HTML because:
-* Some apps will display the browser's default focus styles around the main element
-* When clicking anywhere in the page, focus will return back to the top
+### Focus
+
+The target section must be focusable so that users can then continue to tab through content. Most browsers will do this automatically. However, Internet Explorer 11 does not.
+
+To help skip link work for keyboard and assistive technology users in IE11, add a negative `tabindex` to the main element. 
+
+```html
+<main id="main" tabindex="-1">
+```
```

Comment:
Do we need a little explanation on the tabindex? Something like: "<main> is not a control or a interactive content, so it does not accept focus, but adding tabindex="-1" allows it to accept focus"

Reply:
Good point. Something to keep in mind for the next iteration I think.